common/src/main/java/io/netty5/util/AsciiString.java

```diff
@@ -993,10 +993,10 @@ public AsciiString toUpperCase() {
      * @return a new string with characters {@code <= \\u0020} removed from the beginning and the end.
      */
     public static CharSequence trim(CharSequence c) {
-        if (c instanceof AsciiString) {
+        if (c.getClass() == AsciiString.class) {
```

Comment:
why this change ? Because of https://github.com/netty/netty/pull/12709 ?

Reply:
It shouldn't be a problem (not for the same reasons of #12709) because `AsciiString` is a concrete type which won't interact with the secondary cache field on Klass, see http://cr.openjdk.java.net/~jrose/jvm/checktype-2001.txt for more info how it works (with examples).